nipype/external/cloghandler.py

```diff
@@ -168,11 +158,8 @@ def __init__(self,
         self.maxBytes = maxBytes
         self.backupCount = backupCount
         # Prevent multiple extensions on the lock file (Only handles the normal "*.log" case.)
-        if filename.endswith(".log"):
-            lock_file = filename[:-4]
-        else:
-            lock_file = filename
-        self.stream_lock = open(lock_file + ".lock", "w")
+        self.lock_file = filename + '.lock'
```

Comment:
```suggestion
        self.lock_file = '%s.lock' % filename
```

Again preferring formatting over concatenation (although it is less important this time)